internal/model/label.go

```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
 }
 
 var EnterpriseLabel Label = Label{
-	Name:        "Enterprise (E20)",
-	Description: "This plugin only works on servers with an E20 license.",
+	Name:        "Enterprise (Cloud or E20)",
```

Comment:
```suggestion
	Name:        "Enterprise Edition E20" (Cloud or self-managed),
```

Reply:
I don't know whether the suggested change affects any UI elements.

Reply:
![Screenshot from 2021-02-04 14-14-23](https://user-images.githubusercontent.com/16541325/106897546-5ab79980-66f3-11eb-9b28-04f73a8a3cff.png)

@justinegeffen That is a bit to long :wink: . Can you think of something short?

Reply:
I'm 0/5 on it and happy for it to remain the same. :)